<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>22</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font: 75 22pt &quot;MS Shell Dlg 2&quot;;
padding: 10 px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Morsiphier</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
    background-color: lightgreen;
	color: white;
}
QPushButton{
font: 75 15pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius:20px;
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="encrypt_data">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>400</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border:5px;
border-radius:20px;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhMultiLine</set>
   </property>
   <property name="placeholderText">
    <string>English</string>
   </property>
  </widget>
  <widget class="QPushButton" name="encrypt_btn">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>250</y>
     <width>220</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
    background-color: orange;
	color: white;
}

QPushButton{
background-color: rgb(170, 255, 127);
border-radius:10px;
color:black solid;
}

</string>
   </property>
   <property name="text">
    <string>ENCRYPT -&gt;</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="decrypt_btn">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>440</y>
     <width>220</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
    background-color: orange;
	color: white;
}

QPushButton{
background-color: rgb(170, 255, 127);
border-radius:10px;
color:black solid;
}

</string>
   </property>
   <property name="text">
    <string>&lt;- DECRYPT</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="decrypt_data">
   <property name="geometry">
    <rect>
     <x>699</x>
     <y>80</y>
     <width>400</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border:5px;
border-radius:20px;</string>
   </property>
   <property name="placeholderText">
    <string>Morse Code</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_all_encrypt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>400</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
    background-color: black;
	color: red;
}
QPushButton{
background-color: rgb(203, 203, 203);
color:black;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
border:2px solid red ;
border-radius:20px;
}
</string>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_all_decrypt">
   <property name="geometry">
    <rect>
     <x>699</x>
     <y>640</y>
     <width>400</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
    background-color: black;
	color: red;
}
QPushButton{
background-color: rgb(203, 203, 203);
color:black;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
border:2px solid red ;
border-radius:20px;
}
</string>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
